Classes/HtmlProcessor.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pluswerk\Dashboard;
+
+final class HtmlProcessor
+{
+    private $tableBodyData;
+
+    public function __construct(array $tableBodyDataParam)
+    {
+        $this->tableBodyData = $tableBodyDataParam;
+    }
+
+    private function createOneRow(array $dataItem): string
+    {
+        return '<tr><td>'. $dataItem['name']. '</td><td><a href="https://'. $dataItem['domains'][0]. '">' . $dataItem['domains'][0] . '</a></td></tr>';
+    }
+
+    private function createMultipleRows(array $dataItem): string
+    {
+        $rows = '<tr><td>' . $dataItem['name'] . '</td><td>';
+        foreach ($dataItem['domains'] as $domain) {
```

Comment:
will generate invalid HTML, as the column count depends on configured domains.
for those flexible html, using a div container per Container, with a (li)st (ordered with alphabetical or unordered random would be the better choice.